### ISSUE TITLE
Fix dockerhub automated build

### ISSUE THREAD

**Issue by pavolloffay** [jaegertracing/spark-dependencies#96]
Created from https://github.com/jaegertracing/spark-dependencies/pull/91#issuecomment-652388959

It seems that automated builds are broken - the last build is from 8 months ago https://hub.docker.com/r/jaegertracing/spark-dependencies/builds.

I could not re-initialized the automated build because the docker hub account is not linked with Github - 
https://hub.docker.com/orgs/jaegertracing/settings/linked-accounts. And I cannot link there @jaegertracing organization. 

@yurishkuro could you please link there jaegertracing org? You should be the owner.

**Comment by yurishkuro** [comment]
There's something weird about that repository. All other repos have Permissions tab like this:
![image](https://user-images.githubusercontent.com/3523016/86265222-87219b00-bb91-11ea-9d58-d0f9c318a942.png)

But spark-dependencies repo has Collaborators, where I cannot add anyone
![image](https://user-images.githubusercontent.com/3523016/86265309-9bfe2e80-bb91-11ea-8b56-810519aa99ef.png)

**Comment by pavolloffay** [comment]
Could you please try to link Jaegertracing Github account via https://hub.docker.com/orgs/jaegertracing/settings/linked-accounts?

**Comment by yurishkuro** [comment]
this would link my personal account.

Could we just re-create the repo? I don't understand why it's different from all others and doesn't allow Teams

**Comment by pavolloffay** [comment]
I don't know either. I have created the repo manually and setup the automated build a long time ago.

**Comment by pavolloffay** [comment]
I have tried to create a new repo with automated builds. I could not configure the build on any repository from Jaegertracing github org since docker hub does not have access to Jaegertracing in https://hub.docker.com/orgs/jaegertracing/settings/linked-accounts.

Once I have linked there my personal account and granted access to Jaegertracing I was able to configure the build. 

I am closing this for now as the build is configured correctly.
